.gitlab/build_oci_package.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+
+cd ..
+
+npm pack
```

Comment:
Previous packaging had used the tarballs from the registry rather than repackaging them. We also needed retry logic for this, since we were pretty sure gitlab jobs could not depend on the completion of github jobs.

Packing here _does_ mean it can be done without waiting for the package to appear on the registry, but it also means it's not getting exactly the tarball that's on the registry. Do we accept that tradeoff?

Reply:
I suspect this will never be the case for the _npm_ registry. Sorry if that wasn't clear.

Reply:
One pipeline solves all 🙇  lol.

> Previous packaging had used the tarballs from the registry rather than repackaging them.

Sometimes
*for merge to master and prs*: https://github.com/DataDog/dd-trace-js/blob/master/.github/workflows/test-k8s-lib-injection.yaml#L12 , directly from the code
*for releases/tags*: https://github.com/DataDog/dd-trace-js/blob/master/.github/workflows/release-4.yml#L53 from registry

I'm not a nodejs expert, but does it make a difference? Doesn't the registry return what we send it?